### ISSUE TITLE
Is it still necessary to paste all the lines of code into the android project?

### ISSUE THREAD

**Issue by mariusbolik** [capacitor-community/photoviewer#9]
Hello, I was wondering if it is necessary to paste all the lines of code into the build.gradle files because the code is already in the following file: https://github.com/capacitor-community/photoviewer/blob/main/android/build.gradle

And there's the following command in the build.grade file in the android project root directory:
```
// NOTE: Do not place your application dependencies here; they belong
// in the individual module build.gradle files
```

I'm using capacitor 3.3.3

Edit:
And shouldn't match `implementation "androidx.core:core-ktx:1.6.0"` the by capacitor used core version that's mentioned in the variables.gradle file? So maybe the line should implemented like this: `implementation "androidx.core:core-ktx:$androidxCoreVersion"`

**Comment by jepiqueau** [comment]
@mariusbolik Thanks for using the plugin, i will have a look when updating the plugin to `capacitor@3.3.3` what is required and what is not and update the documentation accordingly. So thanks to mention it to me. When i first develop the plugin it was required.

**Comment by jepiqueau** [comment]
@mariusbolik fix in 1.0.5
